sycl/test/usm/clext.cpp

```diff
@@ -0,0 +1,151 @@
+// RUN: %clang -std=c++11 -fsycl %s -o %t1.out -lstdc++ -lOpenCL -lsycl 
+// RUN: %CPU_RUN_PLACEHOLDER %t1.out -cpu
```

Comment:
It seems that test is expected to work on GPU device at least.
Can we add GPU device type?
HOST?

Reply:
What is the difference between CPU and HOST atm?  If HOST is still OpenCL, sure.

This test would fail on GPU without NEO support for the USM extension, which has been checked in, but not shipped in a .deb yet.

Reply:
> What is the difference between CPU and HOST atm? If HOST is still OpenCL, sure.

Host device do not use OpenCL.

> This test would fail on GPU without NEO support for the USM extension, which has been checked in, but not shipped in a .deb yet.

I suggest skipping the test if USM extension is not supported.

Reply:
Any suggestion on how to do that?